packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/new/new.controller.js

```diff
@@ -0,0 +1,68 @@
+import get from 'lodash/get';
+
+import { VEEAM_TARIFF_DETAILS_URL } from '../backup.constants';
+import { MODES } from '../components/termsOfUse/terms-of-use.constants';
+
+export default class {
+  /* @ngInject */
+  constructor($timeout, $translate, Alerter, datacenterBackupService) {
+    this.$timeout = $timeout;
+    this.$translate = $translate;
+    this.alerter = Alerter;
+    this.datacenterBackupService = datacenterBackupService;
+
+    this.MODES = MODES;
+    this.VEEAM_TARIFF_DETAILS_URL = VEEAM_TARIFF_DETAILS_URL;
+  }
+
+  $onInit() {
+    this.data = {
+      conditionsAccepted: false,
+      selectedOffer: {},
+      orderInProgress: false,
+    };
+  }
+
+  scrollToTop() {
+    this.$timeout(() => {
+      document.getElementById('backupNewMainAlert').scrollIntoView(true);
+    });
```

Comment:
Maybe `$anchorScroll` can be used here no ?

Reply:
Its same, first we need to set the location and then call the scrollTo

        $location.hash(scrollLocation);  
        $anchorScroll();

Reply:
this will add 3 more dependency $location and $anchorScroll apart from $translate